server/main.lua

```diff
@@ -40,17 +40,35 @@ if Config.UseRPName then
     Framework.Object = nil -- free up the memory
 end
 
-local function getPlayerIdentifier(source)
-    local identifier
-    for _, v in ipairs(GetPlayerIdentifiers(source)) do
-        if string.match(v, "license:") then
-            identifier = string.gsub(v, "license:", "")
-            break
-        end
-    end
-    return identifier
+-- local function getPlayerIdentifier(source)
+    -- local identifier
+    -- for _, v in ipairs(GetPlayerIdentifiers(source)) do
+        -- if string.match(v, "license:") then
+            -- identifier = string.gsub(v, "license:", "")
+            -- break
+        -- end
+    -- end
+    -- return identifier
+-- end
+
+function GetPlayerIdentifierFromType(type, source)
```

Comment:
don't modify the function name. keep it `getPlayerIdentifier` that accepts 1 parameter only